src/main/java/io/deephaven/csv/reading/CsvReader.java

```diff
@@ -298,12 +302,15 @@ private static byte check7BitAscii(String what, char c) throws CsvReaderExceptio
     public static final class Result {
         private final long numRows;
         private final String[] columnNames;
-        private final Sink<?>[] columns;
+        private final Object[] columns;
+        private final DataType[] dataTypes;
```

Comment:
Mentioned on google doc, but thoughts about making `Result implement Iterable<ResultColumn>` or something like that?

Reply:
ok done